CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [0.4.1] - 2023-10-04
```

Comment:
The link on this line is broken, check the bottom section of the file how the links are defined.